docs/manual/determinism-checker.tex

```diff
@@ -57,9 +57,10 @@
 \item[\refqualclass{checker/determinism/qual}{NonDet}] indicates
   that the expression might have different values in two different executions on the same machine.
 \item[\refqualclass{checker/determinism/qual}{OrderNonDet}] indicates that
-  a collection (i.e any class that is a subtype of java.util.Collection or java.util.Iterator) or an array will have the same elements in every execution, but in a
+  a collection or an array will have the same elements in every execution, but in a
```

Comment:
Also iterators; I don't think that people usually think of them as "collections".
I don't object to us calling them that, but I think we should clarify our meaning.